CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.28)
+cmake_minimum_required(VERSION 3.22)
```

Comment:
This is the version that comes with ubuntu. We're not using any of the new features so we may as well drop this for compatibility.